configs/containers/docker-ubuntu-gcc-openmpi.yaml

```diff
@@ -4,19 +4,19 @@ spack:
 
   config:
     checksum: false
-    build_jobs: 2
+    build_jobs: 5
```

Comment:
I'm realizing now that I aught to reduce build jobs back to `2` due to test automation.

Reply:
I think so, yes

Reply:
fixed.